components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp

```diff
@@ -4988,6 +4988,21 @@
                                 <td class="leftCol-med labelField"><%=prop.getDisplayName()%>:</td>
                                 <%} %>
                                 <td>
+
+                                <% if (StringUtils.isNotEmpty(prop.getType()) && prop.getType().equalsIgnoreCase("boolean")) {
+                                    String propChecked = "";
+                                    if ( prop.getValue() != null && prop.getValue().equalsIgnoreCase("true")){
```

Comment:
```suggestion
                                    if (prop.getValue() != null && prop.getValue().equalsIgnoreCase("true")){
```

Reply:
This is fixed e983cb8c16466396376a0234e2c1c0aff219187d